### PR TITLE
[UI] Added Big Number component

### PR DESCRIPTION
**Pull request by meghanacosmos**
**Description**
Add Big Number Component. 

Screenshot 2021-12-10 at 3 31 05 AM

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/BigNumber/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { Grid, Box, Typography, Button, CardContent, CardActions, Card } from "@mui/material/";
+
+import PropTypes from "prop-types";
```
Comment (sudo-NithishKarthik) [inline ui/components/BigNumber/index.js:4]: It is really good of you to use prop-types @Meghana-12 , but we have not yet settled on a solution for types. Our intention was to wait for sometime and then, in the future, move to typescript completely. So, I think its better not to waste time incorporating a different style when it is obvious that we have to ditch it.

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]
Just the prop-types can be reverted. Rest looks fine. Thank you @Meghana-12 🎉

File: ui/components/BigNumber/index.js (hunk 1 above)
Comment (Meghana-12) [reply ui/components/BigNumber/index.js:4]: Oh okay! I was getting eslint errors so I added it. Will remove it then!

Comment (sudo-NithishKarthik) [reply ui/components/BigNumber/index.js]: I see. We have to change the eslint config then.

**Review by sudo-NithishKarthik — APPROVED** [review]
LGTM 🎉

**Pull request merged**